tests/python_api/last_batch_policy_testing.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.pipeline import pipeline_def
+from amd.rocal.plugin.pytorch import ROCALAudioIterator
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import random
+import os
+import sys
+import matplotlib.pyplot as plt
+import torch
+import numpy as np
+from parse_config import parse_args
+
+np.set_printoptions(threshold=1000, edgeitems=10000)
+seed = random.SystemRandom().randint(0, 2**32 - 1)
+
+test_case_augmentation_map = {
+    0: "last_batch_policy_FILL_last_batch_padded_True",
```

Comment:
Change the test case name and add more test cases for various LBP combos

Reply:
Need not use the audio golden outputs for testing. Since the label outputs order is known, fetch the labels and compare for each LBP combo

Reply:
TODO : Need to make test suite changes for this.

Reply:
@swetha097 Please work on test suite changes for this